src/index.ts

```diff
@@ -129,7 +129,7 @@ export default async function app(
   // });
 
   app.register(MercuriusGQLUpload, {
-    maxFileSize: 1024 * 1024 * 2,
+    maxFileSize: 1024 * 1024 * 5,
```

Comment:
Let's do something bigger even

```suggestion
    maxFileSize: 1024 * 1024 * 20,
```

Reply:
I thought about that, but the FE is set to 5 anyway?
We can submit this regardless and see if we want to adjust it there?

Reply:
yes, probably good idea to change there as well :)